proto/rill/admin/v1/api.proto

```diff
@@ -654,6 +659,17 @@ message GetOrganizationBillingSubscriptionRequest {
 message GetOrganizationBillingSubscriptionResponse {
   Organization organization = 1;
   Subscription subscription = 2;
+  string billing_portal_url = 3;
+  bool has_payment_method = 4;
+}
+
+message GetOrganizationBillingSessionURLRequest {
+  string org_name = 1;
+  string return_url = 2;
```

Comment:
Consider `redirect_url` or `redirect` – I think that's what we call redirect URLs other places

Reply:
so this is a redirect url actually, page does not redirect after a payment method is entered, this is shown as a simple hyperlink which user has to click to return to our application. See the screenshot below, its the `Return to New Business` link.
![Screenshot 2024-07-11 at 5 02 13 PM](https://github.com/rilldata/rill/assets/11787983/cdd82123-3a1b-450a-b44a-6f15711655af)

Reply:
Okay, then `return_url` makes sense